plutus-metatheory/src/Untyped.lagda.md

```diff
@@ -72,6 +72,13 @@ uglyDATA : DATA → String
 uglyDATA d = "(DATA)"
 
 uglyTmCon : TmCon → String
+
+{-# TERMINATING #-}
+uglyTmConList : (t : TyTag) → ⟦ list t ⟧tag → String
+uglyTmConList t [] = ""
```

Comment:
I guess it's ugly because there are no enclosing `[]` and maybe a trailing comma?

Reply:
Oh right: I was looking for [x] instead of x::[]. In that case it's not ugly at all!